projects/metronome-synth/index.js

```diff
@@ -13,7 +13,8 @@ module.exports = {
         "0x2260FAC5E5542a773Aa44fBCfeDf7C193bc2C599",
         "0xa8b607Aa09B6A2E306F93e74c282Fb13f6A80452",
         "0xc14900dFB1Aa54e7674e1eCf9ce02b3b35157ba5",
-        "0xac3E018457B222d93114458476f3E3416Abbe38F"
+        "0xac3E018457B222d93114458476f3E3416Abbe38F",
+        "0x691Af94cC63B99bd36173eD6Fb1eF5508b2774ec"
```

Comment:
shouldnt this be `0x4Dbe3f01aBe271D3E65432c74851625a8c30Aa7B`

Reply:
apologies - posted the MET asset pool instead of the vaToken - you're correct.